apps/samenvvv/pages/contact.tsx

```diff
@@ -0,0 +1,163 @@
+                  {t('contact.title')}
+                  </Heading>
+                <Text mt={{ sm: 3, md: 3, lg: 5 }} color="primary.50">
+                  {t('contact.fill-form')}
+                </Text>
+              </Box>
+              <VStack alignItems="flex-start" color="primary.50">
+                <Button
+                  as={Link}
+                  isExternal
+                  borderWidth={2}
+                  borderColor="transparent"
+                  variant="ghost"
+                  _hover={{ borderColor: 'primary.400' }}
+                  leftIcon={
+                    <Box as={MdPhone} color="primary.400" size="20px" />
+                  }
+                  href="tel:+31685221308"
+                >
+                  +31-6 85221308
+                </Button>
+                <Button
+                  as={Link}
+                  isExternal
+                  borderWidth={2}
+                  borderColor="transparent"
+                  variant="ghost"
+                  _hover={{ borderColor: 'primary.400' }}
+                  leftIcon={
+                    <Box as={FaWhatsapp} color="primary.400" size="20px" />
+                  }
+                  href="https://api.whatsapp.com/send?phone=31685221308"
+                >
+                  {/* I could not find a suitable message defined here in the common file */}
+                  {t('contact.form.message-label')}
+                </Button>
+                <Button
+                  as={Link}
+                  isExternal
+                  borderWidth={2}
+                  borderColor="transparent"
+                  variant="ghost"
+                  _hover={{ borderColor: 'primary.400' }}
+                  leftIcon={
+                    <Box as={MdEmail} color="primary.400" size="20px" />
+                  }
+                  href="mailto:info@samenvvv.nl"
+                >
+                  info@samenvvv.nl
+                </Button>
+                <Button
+                  as={Link}
+                  isExternal
+                  borderWidth={2}
+                  borderColor="transparent"
+                  variant="ghost"
+                  _hover={{ borderColor: 'primary.400' }}
+                  leftIcon={
+                    <Box as={MdLocationOn} color="primary.400" size="20px" />
+                  }
+                  href="https://goo.gl/maps/E9HaayQnXmphUWtN8"
+                >
+                  Tandersplein 1, 3027 CN <br /> Rotterdam, Netherland
+                </Button>
+              </VStack>
+
+              <SocialButtons items={[]} />
+            </VStack>
+            <Box
+              w={{ base: 'full', lg: '500px' }}
+              bg="white"
+              borderRadius="lg"
+              p={{ base: 8, lg: 16 }}
+              shadow="primary"
+            >
+              <ContactForm />
```

Comment:
```suggestion
              <ContactForm
                onSubmitHandler={handleSubmit}
                isLoading={isLoading}
                isSuccess={isSuccess}
                isError={isError}
            />
```

Since we moved contact form submit mutation out, you should implement them outside of the component, here in this page.

You can see how it's done in [wsvvrijheid](https://github.com/wsvvrijheid/monorepo/blob/main/apps/foundation/pages/contact.tsx)